# resources  
## here is a page where we can share useful resources as we find them
1. background 
    - [NYU: Next-Generation Sequencing Analysis Resources](https://learn.gencore.bio.nyu.edu/)
    - [Fast QC for quality Assessment](https://hbctraining.github.io/Intro-to-ChIPseq/lessons/02_QC_FASTQC.html)
    - [Green Background](https://rpubs.com/linb27/864582)
    - [nextflow variant calling pipeline](https://gencore.bio.nyu.edu/variant-calling-pipeline-gatk4/)
2. software
    - [Bowtie2 homepage](https://bowtie-bio.sourceforge.net/bowtie2/index.shtml)
    - [JBrose Genome Browser](https://jbrowse.org/jbrowse1.html)
4. 1000 genome project
    - [original paper](https://www.nature.com/articles/nature09534)
    - [homepage](https://www.internationalgenome.org/1000-genomes-summary)
4. best practices
    - [A survey of best practices for RNA-seq data analysis](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0881-8?ref=https://githubhelp.com)
    - [Best practices for analyzing microbiomes](https://www.nature.com/articles/s41579-018-0029-9) 
5. paper leads
    - [Google scholar search for papers referencing Bowtie 2](https://scholar.google.com/scholar?oi=bibs&hl=en&cites=4636016874467197548,6107215684849002)